libcontainer/specconv/spec_linux.go

```diff
@@ -333,7 +333,14 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 
 func createLibcontainerMount(cwd string, m specs.Mount) (*configs.Mount, error) {
 	if !filepath.IsAbs(m.Destination) {
-		return nil, fmt.Errorf("mount destination %s not absolute", m.Destination)
+		// Relax validation for backward compatibility
+		oldDest := m.Destination
+		var err error
+		m.Destination, err = filepath.Abs(m.Destination)
```

Comment:
can probably simplify this to `filepath.Join("/", ...)` (like on BuildKit); let me do so